_posts/2016-09-12-schedule.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Schedule Released!"
+authors:
+ - badboy
+ - hoverbear
+---
+
+At long last, a schedule!
+
+We're proud to announce our lineup for RustFest 2016 in Berlin!
+
+[**Click here to view the schedule!**](/schedule)
+
+# Saturday
+
+Starting at 9:00 and going until 19:00, Saturday is full of talks at the Microsoft Atrium!
```

Comment:
We should probably mention that there is an evening gathering at the venue/café after, with buy-your-own-beers, XBoxes and fun. And that the general Berlin Tech community is invited to join for that, just buy the party-ticket and join us!